addons/medical_treatment/ACE_Medical_ReplaceItems.hpp

```diff
@@ -0,0 +1,17 @@
+class GVAR(replaceItems) {
```

Comment:
Don't like the idea of making root-level classes. Would prefer if everything were in a `ace_medical` class.
But just personal preference because I don't like scrolling in config viewer :D

Reply:
Make a better config viewer.

Reply:
Really need a online config viewer that actually is maintained -.-
There are good viewers, but all only show a subset (weapons,vehicles,ammo) and/or are outdated and closed-source.